gammapy/scripts/tests/test_main.py

```diff
@@ -0,0 +1,22 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import pytest
+import sys
+from ..python_api import run_command
+
+
+def test_cmd_main(capsys):
+    with pytest.raises(SystemExit):
+        run_command('', '--help')
+    out, err = capsys.readouterr()
+    assert "Gammapy is a toolbox for high level analysis" in out
+
+
+# on python 2.7 it seems the version info is printed to stderr so we skip the test
```

Comment:
I this function you have tabs -> change to spaces.

Suggest to put this instead, and try to get consistent behaviour on Py 2 & 3. I could look at this tomorrow.
```
def test_cmd_main_version(capsys):
    with pytest.raises(SystemExit):
        run_command('', '--version')
    out, err = capsys.readouterr()
    # Looks like on python 2.7 it seems the version info is printed to stderr
    # whereas on Python 3 it's printed to stdout
    # TODO: understand this!
    txt = out if sys.version_info.major >= 3 else err
    assert 'gammapy' in txt
```